easybuild/framework/easyblock.py

```diff
@@ -1575,7 +1662,68 @@ def sanity_check_step(self, custom_paths=None, custom_commands=None, extension=F
             raise EasyBuildError("Incorrect format for sanity_check_paths (should (only) have %s keys, "
                                  "values should be lists (at least one non-empty)).", ','.join(req_keys))
 
+        commands = self.cfg['sanity_check_commands']
+        if not commands:
+            if custom_commands:
+                commands = custom_commands
+                self.log.info("Using customised sanity check commands: %s" % commands)
+            else:
+                commands = []
+                self.log.info("Using specified sanity check commands: %s" % commands)
+
+        for i, command in enumerate(commands):
+            # set command to default. This allows for config files with
+            # non-tuple commands
+            if not isinstance(command, tuple):
+                self.log.debug("Setting sanity check command to default")
+                command = (None, None)
```

Comment:
no, the 1st `None` is important, it basically results in using `self.name.lower()` as default command (see below)